com.unity.formats.alembic/Documentation~/TableOfContents.md

```diff
@@ -16,10 +16,13 @@
 * [Working with Curves](curves.md)
 * [Applying motion blur and shaders](motion-vectors.md)
 * [Window reference](reference.md)
```

Comment:
I wonder whether we can improve the order of the sub topics: group the import and export topics? Not sure whether that makes sense, because I don't know Alembic well enough.

Reply:
What would you think of something like this (and the table in reference.md could reflect it):

* [Window reference](reference.md)
    * [Alembic Import Settings window](ref_Importer.md)
        * [Alembic Stream Player component](ref_StreamPlayer.md)
    * [Alembic Curves component](ref_Curves.md)
        * [Alembic Curves Renderer component](ref_CurvesRenderer.md)
    * [Alembic Custom Data component](ref_CustomData.md)
    * [Alembic Point Cloud component](ref_PointCloud.md)
        * [Alembic Point Renderer component](ref_PointRenderer.md)
    * [Alembic Exporter component](ref_Exporter.md)
    * [Alembic Clip Recorder](ref_Recorder.md)
        * [Alembic Shot Asset clip settings](ref_Shot.md)

Reply:
That looks really good! I will check the content model to see if we can have that many indents and get back to you.

Reply:
Looks like 3 level in the TOC is fine. I'm wondering about where Alembic Shot Asset clip settings belongs. I suspect it's related to Alembic Exporter component, not Alembic Clip Recorder. If it affects both, I would move it to level 2.

Reply:
From what I understand, the `Alembic Shot Asset clip settings` is the setting that describes an alembic shot in a Timeline Alembic Track. I would not put them under Exporter, because they are not directly related (unlike the importer and the stream player)

I think we could put the `Alembic Clip Recorder` and the `Alembic Shot Asset clip settings` at level 2, or we could put them at level 3 under a `Timeline` subtitle, for example:
* [Window reference](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/523/reference.md)
    * ...
    * Timeline
        * [Alembic Clip Recorder](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/523/ref_Recorder.md)
        * [Alembic Shot Asset clip settings](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/523/ref_Shot.md)

Reply:
I would leave Alembic Clip Recorder on level 2 with the Exporter; it exists as a window, where you can use it without a Timeline, and can be added to the Timeline by adding a Recorder track.  The person looking for it would be interested in exporting. (I think I need to update the topic on Alembic Clip Recorder.)
I just reread the existing documentation: the Alembic Shot Asset Clip component is 
 mentioned in the topic "Play back using an Alembic Shot clip", and nowhere else. So if you created a Timeline category, it could go there, but so would Alembic Shot Asset component..

Reply:
Sounds good! :)